src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -15,6 +15,7 @@
     <!-- Split semicolon separated lists -->
     <WithCategoriesItems Include="$(WithCategories)" />
     <WithoutCategoriesItems Include="$(WithoutCategories)" />
+    <DefaultNoCategories Include="$(DefaultNoCategories)" />
```

Comment:
nit: I realize this item is already used below but from a consistency point of view I would have called this DefaultNoCategoriesItems like the other places.